recipes/geospatial/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "geospatial" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/geospatial-{{ version }}.tar.gz
+  sha256: 93edf988a6606e21f8b13251c2e8c6c073ef5b8f7b96d51ec38ef8b0a00e4587
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
```

You list 3.5 upstream. Are you sure that this is correct?
```suggestion
    - python >=3.5
```

Ref: https://github.com/giswqs/geospatial/blob/73ca0ab34e20d5790b92ec3364389323057255e2/setup.py#L30

---

I have updated the upstream to require Python >=3.7